scripts/generate_onnx_stubs.py

```diff
@@ -0,0 +1,695 @@
+import inspect
+import itertools
+import os
+import sys
+import re
+
```

Comment:
Is this file still being used?

Reply:
I try to replace this draft with the module. will delete it when I'm sure I didn't miss anything